src/main/java/edu/kis/powp/jobs2d/features/driverTransofrmation/modifiers/YAxisMovingModifier.java

```diff
@@ -0,0 +1,16 @@
+package edu.kis.powp.jobs2d.features.driverTransofrmation.modifiers;
+
+import edu.kis.powp.jobs2d.features.driverTransofrmation.TransformationModifier;
+
+public class YAxisMovingModifier implements TransformationModifier {
+    private final int shiftY;
+
+    public YAxisMovingModifier(int shiftY) {
+        this.shiftY = shiftY;
+    }
+
+    @Override
+    public int[] modify(int x, int y) {
+        return new int[]{x, (y + shiftY)};
+    }
+}
```

Comment:
OK, but  XAxisMovingModifier and  YAxisMovingModifier could be implemented as a single class.